packaging/windows/BUILD.txt

```diff
@@ -0,0 +1,120 @@
+How to make a darktable windows installer (64 bit only):
```

Comment:
Makes sense. I saw the doc/windows.txt actually, it seems to me describing how to do a cross-compile. What would be the best location of the merged file: /doc/windows.txt or the packaging/windows/BUILD.txt?